setup.py

```diff
@@ -9,13 +9,39 @@
 This script is interactive and will prompt you for various inputs.
 """
 
+
+def filter_lines(file_path, excluded_ranges):
+    with open(file_path, 'r') as file:
+        lines = file.readlines()
+
+    new_lines = []
+    current_line = 1
+    for line in lines:
+        exclude = any(start <= current_line <= end for start, end in excluded_ranges)
+        if not exclude:
+            new_lines.append(line)
+        current_line += 1
+
+    with open(file_path, 'w') as file:
+        file.writelines(new_lines)
+
+
 if __name__ == "__main__":
+
     repo_name = input("What is the name of your plugin (full name with spaces)? ")
 
     snake_name = "_".join(repo_name.lower().split(" "))
 
     os.rename("my_plugin.py", f"{snake_name}.py")
 
+    # Pipeline file's line ranges to remove
+    # This must reflect eventual main.yml changes
+    excluded_line_ranges = [(24, 35), (37, 40)]
```

Comment:
Isn't it better if we check by comments instead of by line ranges?

Reply:
Why? What comment we should check for? Specific one? All the comments? What if we want to add other comments not related to the erasure task?

IMHO this was the simplest solution to the problem; less lines of code and not to complicate to read/understand.

Reply:
Ok chill bro, don't kill my family plz

Reply:
I know your address! 🔪 🚿 😱 🩸